### ISSUE TITLE
Publish build and test reports

### ISSUE THREAD

**Issue by wenkokke** [plfa/plfa.github.io#685]
The build and tests generate reports:

- **Shake**: already generated, see: [Makefile#L35](https://github.com/plfa/plfa.github.io/blob/c0464e4e31679c59291ec6198008d8f9b28617e3/Makefile#L35)
- **HTMLProofer**: see [HTMLProofer: Reporting](https://github.com/gjtorikian/html-proofer#reporting)
- **html-validate**
- **EPUBCheck**: see [EPUBCheck: Getting a report in a structured format](https://www.w3.org/publishing/epubcheck/docs/cli/#getting-a-report-in-a-structured-format)
- **Daisy Ace**: see [Daisy Ace: HTML Report](https://daisy.github.io/ace/docs/report-html/)
- **Pandoc**: log warnings in `runPandoc`

These reports should be published as part of the web version of the book.
